pkg/resources/validation/cluster/provisioningcluster.go

```diff
@@ -23,6 +23,7 @@ import (
 const globalNamespace = "cattle-global-data"
 
 var mgmtNameRegex = regexp.MustCompile("^c-[a-z0-9]{5}$")
+var fleetNameRegex = regexp.MustCompile("^[-a-z0-9]*$")
```

Comment:
So is "-----------------" a fine cluster name? And could + be used instead of "*" since we probably don't want an empty name?